misc/demo_embed_asm.py

```diff
@@ -0,0 +1,16 @@
+import taichi as ti
```

Comment:
Could you rename this file to `misc/demo_opengl_embed_asm.py` or simply make it a test?

Reply:
Done, using `ti.extension.extfunc`, we may also write a extfunc test on CPU later.